### PR TITLE
Case display name in downloaded PDF

### PR DESCRIPTION
**Pull request by northwestwitch**
When the PDF download is called the downloaded file contains the case display name and the current date

### CODE REVIEW

**Comment by dnil** [comment]
Hm, what's the status of this? 😊

**Comment by northwestwitch** [comment]
> Hm, what's the status of this? 😊

I could have written this in another life, for all I remember about it!

**Comment by northwestwitch** [comment]
Should be reviewed I guess

File: chanjo_report/server/blueprints/report/views.py [hunk 1]
```diff
@@ -101,7 +103,12 @@ def pdf():
 
     # check if the request is to download the file right away
     if 'dl' in request.args:
```
Comment (dnil) [inline chanjo_report/server/blueprints/report/views.py:105]: Shouldn't this be data_dict now rather to cater for both GET and POST?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Change log
 
+
+## 3.3.0 (2020-04-23)
```
Comment (dnil) [inline CHANGELOG.md:4]: Ahem, why is that 3.3.0 and the version 4.3.0?

Comment (northwestwitch) [reply CHANGELOG.md:4]: Ahem.. Ok I'll write the usual X.X.X version and we can update it in your new PR then!

File: chanjo_report/server/blueprints/report/views.py (hunk 1 above)
Comment (northwestwitch) [reply chanjo_report/server/blueprints/report/views.py:105]: Good point, I'll change it!

File: setup.py [hunk 3]
```diff
@@ -45,7 +45,7 @@ def run_tests(self):
 
 setup(name='chanjo-report',
       # versions should comply with PEP440
-      version='4.2.0',
+      version='4.3.0',
```
Comment (northwestwitch) [inline setup.py:48]: Ok I'll leave 4.2 here

Comment (northwestwitch) [reply setup.py:48]: Actually maybe I'll set both to 4.3, makes more sense, since something actually changed from the previous version

**Comment by northwestwitch** [comment]
Changed according to your suggestions, thanks for the review @dnil ! I think I'll merge now, then if anything is wrong we can fix it in your PR 😉

**Pull request merged**